vue-frontend/src/components/services/ContactUsToday.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <section
+    class="lg:tw-relative tw-mt-10 lg:tw-mb-[300px] tw-bg-gradient-to-t tw-from-[#FF835B]/[0.08] tw-to-[#F2709C]/[0.08] tw-pt-[2rem] tw-pb-[2rem] lg:tw-bg-none"
+  >
+    <div class="tw-w-full tw-h-full">
+      <img
+        v-if="width > 991"
+        :src="contactbgImages[3]"
+        :srcset="`${contactbgImages[1]} 400w, ${contactbgImages[1]} 800w, ${contactbgImages[2]} 1200w, ${contactbgImages[3]} 1600w`"
+        loading="lazy"
+        class="tw-hidden lg:tw-block tw-absolute tw-top-[217px] tw-left-0 tw--z-[1] tw-object-cover"
+        alt="background-image"
+      />
+    </div>
+    <div
+      class="tw-container tw-flex tw-flex-col lg:tw-flex-row-reverse tw-items-center lg:tw-gap-[1.5rem]"
+    >
+      <div class="tw-flex tw-flex-col lg:tw-w-[59%] xl:tw-w-[50%]">
+        <h2
+          class="tw-font-inter-regular tw-text-[1.875rem] md:tw-text-[2.5rem] lg:tw-text-[3rem] tw-leading-[2.813rem] md:tw-leading-[3.75rem] lg:tw-leading-[4.5rem] tw-text-center lg:tw-text-left tw-text-black-core/[0.87]"
+        >
+          Do you want to
+          <span
+            class="tw-font-inter-bold lg:tw-text-[3.438rem] lg:tw-leading-[5.156rem]"
+            >provide your customers</span
+          >
+          with value at
+          <span
+            class="tw-font-inter-bold lg:tw-text-[3.438rem] lg:tw-leading-[5.156rem]"
+            >their fingertips?</span
+          >
+        </h2>
+        <div class="tw-hidden lg:tw-block tw-mt-[1.5rem]">
+          <router-link
+            class="tw-relative tw-bg-gradient-[1deg] tw-pb-[12px] tw-text-[2.5rem] tw-leading-[3.75rem] v2-canopas-gradient-text tw-font-inter-bold gradient-underline tw-no-underline tw-cursor-pointer after:tw-absolute after:tw-bottom-0 after:tw-left-0 after:tw-w-full after:tw-h-[5px] after:tw-bg-gradient-to-l after:tw-to-[#F2709C] after:tw-from-[#FF835B]"
+            :to="contactURL"
+            @click.native="mixpanel.track('tap_services_contact')"
+            ><span> Contact Us Today</span></router-link
+          >
+        </div>
+      </div>
+      <div
+        class="tw-flex tw-flex-col tw-items-center lg:tw-w-[41%] xl:tw-w-[50%]"
+      >
+        <aspect-ratio height="30%">
+          <img
+            class="tw-h-full tw-w-full tw-object-cover"
+            :src="contactImages[0]"
+            :srcset="`${contactImages[0]} 400w, ${contactImages[1]} 800w, ${contactImages[2]} 1200w, ${contactImages[2]} 1400w`"
```

Comment:
The ideal image width in 30% of the full window width, I think no need to take images> 800 widths in the source set.
you can check the max render the width of images.